toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -661,3 +661,51 @@ func extractWorkerTar(chroot string, workerTar string) (err error) {
 	_, _, err = shell.Execute("tar", "-I", gzipTool, "-xf", workerTar, "-C", chroot)
 	return
 }
+
+type ChrootInterface interface {
+	RootDir() string
+	Run(toRun func() error) error
+	UnsafeRun(toRun func() error) error
+	AddFiles(filesToCopy ...FileToCopy) error
+}
+
+// DummyChroot is a placeholder that implements ChrootInterface.
+type DummyChroot struct {
+}
+
+func (d DummyChroot) RootDir() string {
+	return "/"
+}
+
+func (d DummyChroot) Initialize(tarPath string, extraDirectories []string, extraMountPoints []*MountPoint) error {
+	// No operation for the dummy type
+	return nil
+}
+
+func (d DummyChroot) Run(toRun func() error) (err error) {
+	// Only execute the function, no chroot operations
+	return toRun()
+}
+
+func (d DummyChroot) UnsafeRun(toRun func() error) (err error) {
+	return toRun()
+}
+
+func (d DummyChroot) AddFiles(filesToCopy ...FileToCopy) (err error) {
```

Comment:
This code seems to be identical to the `Chroot` variant. It might be goodness to figure out a way for them to share code.